File: solutions/0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}

        for i in range (len(nums)):
            if (target - nums[i] in hashMap):
                return [i, hashMap[target-nums[i]]]
            hashMap[nums[i]] = i

File: solutions/0001-two-sum/solution.ts
function twoSum(nums: number[], target: number): number[] {
    let map = new Map<number, number>()

    for (let i = 0; i < nums.length; i++) {
        if (map.has(nums[i])) return [i, map.get(nums[i])]
        map.set(target - nums[i], i)
    }

    return []
};
